### PR TITLE
Add nbconvert to RTD requirements

### PR DESCRIPTION
**Pull request by larrybradley**

### CODE REVIEW

**Comment by pllim** [comment]
I don't think you need this.

**Comment by pllim** [comment]
Unless this is to fix the warnings?
```
/home/docs/checkouts/readthedocs.org/user_builds/astroimtools/checkouts/latest/docs/astroimtools/cutouts.rst:16: WARNING: unknown document: notebooks/make_cutouts
/home/docs/checkouts/readthedocs.org/user_builds/astroimtools/checkouts/latest/docs/astroimtools/cutouts.rst:17: WARNING: unknown document: notebooks/make_rgb
/home/docs/checkouts/readthedocs.org/user_builds/astroimtools/checkouts/latest/docs/astroimtools/scripts.rst:11: WARNING: Could not lex literal_block as "python3". Highlighting skipped.
/home/docs/checkouts/readthedocs.org/user_builds/astroimtools/checkouts/latest/docs/astroimtools/scripts.rst:17: WARNING: Could not lex literal_block as "python3". Highlighting skipped.
/home/docs/checkouts/readthedocs.org/user_builds/astroimtools/checkouts/latest/docs/astroimtools/scripts.rst:27: WARNING: Could not lex literal_block as "python3". Highlighting skipped.
/home/docs/checkouts/readthedocs.org/user_builds/astroimtools/checkouts/latest/docs/astroimtools/scripts.rst:37: WARNING: Could not lex literal_block as "python3". Highlighting skipped.
/home/docs/checkouts/readthedocs.org/user_builds/astroimtools/checkouts/latest/docs/astroimtools/scripts.rst:49: WARNING: Could not lex literal_block as "python3". Highlighting skipped.
```

**Comment by larrybradley** [comment]
Yes, but look at https://astroimtools.readthedocs.io/en/latest/astroimtools/cutouts.html
`nbconvert` is needed to make those cutout notebooks.

**Comment by pllim** [comment]
Ah, okay. Then we can add back `nbconvert`, but why `jupyter`?

**Comment by larrybradley** [comment]
I think `nbconvert` requires `jupyter`, but I'll try it without and hope it will install it itself if needed.  Thanks.

**Comment by pllim** [comment]
AFAIK never had to specify `jupyter` explicitly before. 🤷‍♀️

File: requirements.txt [hunk 1]
```diff
@@ -5,4 +5,4 @@ astropy
 photutils
 Sphinx
 sphinx-astropy
-nbconvert
+nbcovert
```
Comment (pllim) [inline requirements.txt]:
```suggestion
nbconvert
```

File: requirements.txt [hunk 2]
```diff
@@ -5,3 +5,4 @@ astropy
 photutils
 Sphinx
 sphinx-astropy
+nbcovert
```
Comment (pllim) [inline requirements.txt]:
```suggestion
nbconvert
```

Comment (pllim) [reply requirements.txt]: `nbcovert` sounds like something we might use in CSI... 😏

**Comment by larrybradley** [comment]
I just noticed that the lupton RGB notebook needs to be removed too.

**Comment by pllim** [comment]
Feel free. RTD passed, so I am gonna stop bothering you now...

**Comment by larrybradley** [comment]
OK, will do!

**Comment by larrybradley** [comment]
I'm going to go over all the docs and code again before making a new release.  Anything you want to add before a new release?

**Comment by pllim** [comment]
> Anything you want to add before a new release?

Not for this release, but I think there needs to be a campaign to retire this package altogether...

**Pull request merged**